maven/maven.bzl

```diff
@@ -238,11 +238,11 @@ def _maven_jvm_artifact(artifact_spec, name, visibility, deps = [], exports = []
     maven_target = "@%s//%s:%s" % (artifact.maven_target_name, _DOWNLOAD_PREFIX, artifact.path)
     import_target = artifact.maven_target_name + "_import"
     target_name = name if name else artifact.third_party_target_name
-    exports = exports + deps  # A temporary hack since the existing third_party artifacts use exports instead of deps.
+    deps = exports + deps  # A temporary hack since the existing some artifacts use exports instead of deps.
```

Comment:
typo, `the existing some artifacts`

Reply:
Gah - that was a hack related to the old, internal version of this code.  Since we're generating the mvn_jvm_artifact statements, we control what goes in what, so there is nothing using exports unless someone hacked something that's very very wrong.  I'm going to get rid of it. I should have before doing this PR.